### ISSUE TITLE
Handling connectivity issues

### ISSUE THREAD

**Issue by thatoddmailbox** [MyHomeworkSpace/client#19]
Right now, if you lose connection to the Internet, it looks like all your homework is missing, which is bad.

**Comment by nywillb** [comment]
Yeah. I just got really scared because I thought my all my homework had been deleted.

**Comment by nywillb** [comment]
We should probably just put a "Hey... It seems like you are not connected to the internet!" with a frowning cloud.
